pallets/assets-manager/src/lib.rs

```diff
@@ -33,7 +33,8 @@ pub mod pallet {
     use xcm::prelude::{Abstract, AssetId, Concrete, GeneralIndex, PalletInstance, Parachain, X3};
     use xcm_executor::traits::Convert;
 
-    pub(crate) const STORAGE_VERSION: StorageVersion = StorageVersion::new(2);
+    pub(crate) const STORAGE_VERSION_: usize = 2;
+    pub(crate) const STORAGE_VERSION: StorageVersion = StorageVersion::new(STORAGE_VERSION_ as u16);
```

Comment:
This pallet is going to be removed, right?

Reply:
doesn't hurt to remove wrong code